container_files/Containerfile.services

```diff
@@ -1,114 +1,178 @@
+# backendbuilder
 FROM registry.access.redhat.com/ubi9/ubi:latest as backendbuilder
 
-ARG RUST_VERSION="1.77.2"
+USER root
+
+RUN dnf install -y gcc openssl openssl-devel cmake gcc-c++ git unzip cyrus-sasl-devel
+
+RUN groupadd -r -g 1001 trustuser \
```

Comment:
Why is it necessary to make the **builder** part of the container image so complex? I do understand that for the runtime container it might make sense, but for parts of the builder?

Reply:
If we create as a root all the files we can't use with normal user permissions in other layers.